raiden/transfer/state.py

```diff
@@ -183,6 +178,20 @@ def __post_init__(self) -> None:
             raise ValueError("node_address must be an address instance")
 
 
+@dataclass
+class RouteState(State):
+    """ A possible route for a payment to a given target. """
+
+    # TODO: Add timestamp
+    route: List[Address]
```

Comment:
As discussed yesterday, it makes alot of sense for me to have this as a list of `HopState`s. The member can be renamed to `hops`. If this is done, `route.forward_channel_id` becomes:
```
next_hop = route.hops.next()
next_hop.channel_identifier
```

To me, this makes it easier to understand the code. As i am currently confused about the relation between `RouteState` and `HopState`, this could make it easier to understand that a route is a list of hops. where [0] is the current node and [1] is the next hop... etc.

Reply:
I see your point, but we need to think to think a bit about that because we currently only know the channel id for channels the node is part of. So then the `HopState`s `channel_identifier` would become optional, which might or might not cause some trouble.

Let's look at this after this PR.